libmamba/include/mamba/core/util_os.hpp

```diff
@@ -12,6 +12,11 @@
 
 #include "mamba/core/fsutil.hpp"
 
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#endif
```

Comment:
Is it be possible to avoid including <Windows.h> in a "public" header? I'm a bit ocncerned with all the Windows headers this pulls in.

Reply:
No, I had to add it because of the usage of `DWORD`. Note that the reason it was not visible before is because there is no test before this PR including this header alone. This means that the other headers already include `windows.h`.
Note that `WIN32_LEAN_AND_MEAN` should drastically reduce the issue, but the only way to remove that would be to not use `DWORD` OR to use our definition of `DWORD`, which is a possibility, like they did in Boost.WinAPI for example.

Reply:
Used an alias instead.